hoot-core/src/main/cpp/hoot/core/visitors/RemoveInvalidMultilineStringMembersVisitor.cpp

```diff
@@ -60,7 +60,7 @@ void RemoveInvalidMultilineStringMembersVisitor::visit(const ElementPtr &e)
         if (_map->getElement(it->getElementId())->getTags().getInformationCount() > 0)
           return;
       }
-      const double expansion = r->getCircularError();
+      const double expansion = r->getCircularError() / 2.0;
```

Comment:
Yeah it should help, it reduces the size of the envelope I have additional code in another review that is almost ready that should do an even better job.